### PR TITLE
Changes to the emails for the ReplyTo text, and fixing time zone issue

### PR DESCRIPTION
**Pull request by dborgen2**
## What’s changing?

Additional text in both Type 1 and Type 2 emails for how the user can get help with their match.
Fixing issue with time zones changing the date of the match.

## Why?

Closes NAC-1613 and NAC-1666

## This PR has:

- [x] **JIRA ticket(s)** (_Must be listed in PR request and branch name should follow convention NAC-123-description_)<description>

- [x] **Security Related Changes** (_Security Tab questions have been answered on JIRA ticket(s)_)

- [x] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [_] **Documented API changes** (_required on public classes and methods, and elsewhere as appropriate_)
    - [ ] External API documentation - OpenApi docs and Generated Docs (e.g. https://github.com/18F/piipan/tree/dev/docs/openapi) 
    - [ ] Internal API documentation (e.g. https://github.com/18F/piipan/tree/dev/metrics/docs/openapi)   

- [x] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [x] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [x] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)
    - [x] Have IaC scripts run successfully against an existing deployed environment?
    - [ ] Have IaC scripts run successfully against a newly deployed environment?
    - [ ] Do the IaC changes require deleting existing resources?

## Anything else?

_Add other details that are helpful for review here_

## Code Review Checklist:

- [x] Automated checks all pass? (code builds successfully, tests pass, test coverage meets/exceeds threshold, etc.) 
- [x] Have all requirements and Acceptance Criteria been met?
- **If there are code changes**
    - [x] Is there no unnecessary complexity? Code is readable, maintainable, etc.
    - [x] Do the changes follow SOLID principles?
    - [x] Are the changes robust? Concurrency considered? Performance? Security (e.g. no sql injections)?
    - [x] Are newly added dependencies needed and appropriate? Are dependency files (e.g. packages.json & packages.lock.json) updated?
    - [x] For any new projects/solutions, have they been added to code coverage files?
    - [x] Public classes & methods are documented    
- **If there are API changes**
    - [ ] Does API make sense? Useful and concise?
    - [ ] Are existing API interfaces/contracts intact? i.e. no breaking changes
    - [ ] Are API tests included?
    - [ ] Are API responses appropriate and documented? (200s, 201, 404s, etc.)
    - [ ] Are internals kept private and not leaked into the API?
    - [ ] External API documentation - OpenApi docs and Generated Docs 
    - [ ] Internal API documentation    
- **If there are IAC changes**
    - [x] IAC Scripts are readable & maintainable
    - [x] IAC Scripts are documented
    - [x] IAC scripts pass required checks
    - [x] IAC scripts were run successfully by author
- **General Logging Questions**
    - [ ] Do new capabilities (or changed capabilities) include logging?
    - [ ] Is PII not logged?
- **General Error Handling Questions**
    - [ ] Has exception handling been added for new capabilities (or changed capabilities)?
    - [ ] Are errors/exceptions properly communicated to end-users? Were there requirements for user facing error messages?
    - [ ] Are errors/exceptions logged?
    - [ ] Is PII kept out of error/exception logging?
- **If there are Developer Documentation Changes**
    - [ ] General Project documentation (under piipan\docs. e.g. iac.md, architecture.md, etc.)
    - [ ] Github documentation additions/changes (Readmes)
- **General automated test questions**
    - [ ] End-to-End Tests provided (if code implements new user story)
    - [ ] Are tests sufficient?
    - [ ] Corner cases considered?
    - [ ] Are performance tests needed?

### CODE REVIEW

File: match/src/Piipan.Match/Piipan.Match.Core/Services/MatchResNotifyService.cs [hunk 1]
```diff
@@ -64,8 +65,9 @@ public async Task SendNotification(MatchDetailsDto matchResRecordBeforeUpdate, M
                 InitState = initState?.State,
                 MatchingState = matchingState?.State,
                 MatchingUrl = $"{queryToolUrl}/match/{matchResRecordBeforeUpdate.MatchId}",
-                InitialActionBy = DateTime.Now.AddDays(10),
-                CreateDate = matchResRecordAfterUpdate.CreatedAt
+                InitialActionBy = DateTime.UtcNow.ToEasternTime().AddDays(10),
```
Comment (bgfoshay) [inline match/src/Piipan.Match/Piipan.Match.Core/Services/MatchResNotifyService.cs:68]: Thought: We might consider adding a comment to this line detailing why it's converting to Eastern Time, maybe even adding a reference to the JIRA ticket number.

File: match/src/Piipan.Match/Piipan.Match.Core/Services/MatchResNotifyService.cs [hunk 2]
```diff
@@ -64,8 +65,9 @@ public async Task SendNotification(MatchDetailsDto matchResRecordBeforeUpdate, M
...
+                CreateDate = matchResRecordAfterUpdate.CreatedAt,
+                ReplyToEmail = Environment.GetEnvironmentVariable("SmtpReplyToEmail")
```
Comment (bgfoshay) [inline match/src/Piipan.Match/Piipan.Match.Core/Services/MatchResNotifyService.cs:70]: Thought/Nitpick: Is there an opportunity to pull this out of SendNotification so that we're not re-retrieving the environment variable every time SendNotification is called?

File: notifications/src/Piipan.Notification.Common/Templates/MatchEmail.cshtml [hunk 3]
```diff
@@ -42,7 +42,8 @@
     </style>
 </head>
 <body>
-    <p>This is an automated notification to inform you that a NAC match has been found with your state.</p>
+    <p>This is an automated notification and we are unable to accept replies. If you need additional assistance, please contact us at <a href="mailto:@(Model.ReplyToEmail)?subject=NAC%20Match%20@(Model.MatchId)">@Model.ReplyToEmail</a>.</p>
+    <p>A NAC match has been found with your state.</p>
```
Comment (bgfoshay) [inline notifications/src/Piipan.Notification.Common/Templates/MatchEmail.cshtml:46]: Question: Do we need to convert the InitialActionBy.ToShortDateString() call below to Eastern Time? (line 54)

Comment (dborgen2) [reply notifications/src/Piipan.Notification.Common/Templates/MatchEmail.cshtml:46]: No, shouldn't need to. I converted it in the MatchEventService, so it is still in Eastern Time at this point.

File: match/src/Piipan.Match/Piipan.Match.Core/Services/MatchResNotifyService.cs (hunk 2 above)
Comment (dborgen2) [reply match/src/Piipan.Match/Piipan.Match.Core/Services/MatchResNotifyService.cs:70]: Yea, that's a good idea. I'll pull it out and perform the action in the constructor.

**Review by bgfoshay — APPROVED** [review]
👍
